src/main/java/core/basesyntax/service/impl/DataConverterImpl.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.OperationType;
+import core.basesyntax.model.ShopOperation;
+import core.basesyntax.service.DataConverter;
+import java.util.ArrayList;
+import java.util.List;
+
+public class DataConverterImpl implements DataConverter {
+    private static final int LIST_START = 1;
+    private static final int OPERATION_INDEX = 0;
+    private static final int FRUIT_INDEX = 1;
+    private static final int QUANTITY_INDEX = 2;
+    private static final String COMMA = ",";
+
+    @Override
+    public List<ShopOperation> convertToOperation(List<String> reportList) {
+        List<ShopOperation> shopOperationList = new ArrayList<>();
+        for (String line : reportList.subList(LIST_START, reportList.size())) {
```

Comment:
The method assumes that the input list `reportList` has at least one line to skip (likely a header). Ensure that the input list is validated to prevent `IndexOutOfBoundsException` if the list is empty or has fewer lines than expected.